superflore/generators/bitbake/run.py

```diff
@@ -31,38 +33,35 @@
 from superflore.utils import info
 from superflore.utils import load_pr
 from superflore.utils import ok
+from superflore.utils import ros2_distros
 from superflore.utils import save_pr
 from superflore.utils import url_to_repo_org
 from superflore.utils import warn
 
-# Modify if a new distro is added
-active_distros = ['indigo', 'kinetic', 'lunar']
-
-
 def main():
-    preserve_existing = True
+    os.environ["ROS_OS_OVERRIDE"] = "oe"
     overlay = None
+    preserve_existing = True
     parser = get_parser('Deploy ROS packages into Yocto Linux')
+    parser.add_argument(
+        '--skip-keys',
+        nargs='+',
+        help='The specified keys will be ignored'
```

Comment:
What does 'keys' mean here? Can this be added to the `get_parser` function?

Reply:
It's similar to [rosdep's skip-keys](http://docs.ros.org/independent/api/rosdep/html/commands.html), where we want to explicitly skip some packages and not adding them as dependencies.
I'll improve the help message. Not sure if it makes sense to ebuild/Gentoo so I didn't move it to get_parser.

Reply:
Yeah, that sounds useful to me. If you could please file an issue so I remember to centralize this after we get the initial open embedded support in that would be great!

Reply:
Filed https://github.com/ros-infrastructure/superflore/issues/170 for centralizing the option.